toolchain/mfc/bench.py

```diff
@@ -1,6 +1,6 @@
 import os, sys, uuid, subprocess, dataclasses
 
-import rich.table
+import rich.table, typing
```

Comment:
Nitpick but `typing` should be imported on the first line with the other Python standard library dependencies.

Reply:
Done